javascriptv3/example_code/bedrock-agent/tests/bedrock-agent-actions.integration.test.js

```diff
@@ -0,0 +1,149 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { afterAll, beforeAll, describe, expect, it } from 'vitest';
+
+import { getRandomAlphanumericString } from '@aws-sdk-examples/libs/utils/util-string.js';
+import { wait } from '@aws-sdk-examples/libs/utils/util-timers.js';
+import { log } from '@aws-sdk-examples/libs/utils/util-log.js';
+
+import {
+    CreateRoleCommand, DeleteRoleCommand,
+    DeleteRolePolicyCommand,
+    IAMClient,
+    PutRolePolicyCommand,
+    waitUntilRoleExists
+} from '@aws-sdk/client-iam';
+
+import {createAgent} from '../actions/create-agent.js';
+import {listAgentsWithPaginator} from '../actions/list-agents.js';
+import {getAgent} from '../actions/get-agent.js';
+import {deleteAgent} from '../actions/delete-agent.js';
+
+/**
+ * @typedef {Object} Agent
+ * @property {string} agentId
+ * @property {string} agentName
+ * @property {string} agentStatus
+ */
+
+const REGION = 'us-east-1';
+const AGENT_POLICY_NAME = 'can-invoke-bedrock-model';
+const FOUNDATION_MODEL = 'anthropic.claude-v2';
+
+const postfix = getRandomAlphanumericString(8);
+const testAgentName = `test-agent-${postfix}`;
+const agentRoleName = `AmazonBedrockExecutionRoleForAgents_${postfix}`;
+let agentResourceRoleArn;
+
+describe('create-list-get-delete', () => {
+    const iam = new IAMClient();
+
+    // Ideally these should be separate tests, but since we're creating
+    // real AWS resources, this saves on testing time and cost.
+    it('should-successfully-complete', async () => {
+        await expect(async () => {
```

Comment:
Unnecessary expect wrapper. The awaits below will throw if the promise is rejected, and that will fail the test.

Reply:
Only added to satisfy my local linter complaining about a test without expect statement. Happy to remove.

Reply:
Yes please. Might be a linter setting worth disabling. Forcing expects seems like handcuffs to me.